bazaarvoice/emodb
# PD-194381: fix the SonarQube Security Hotspots

## Github Issue #

[PD-194381](https://bazaarvoice.atlassian.net/browse/PD-194381)

## What Are We Doing Here?

The SonarQube Security Hotspots fixing

## How to Test and Verify

1. Check out this PR
2. Run Command: `mvn clean install -DskipTests=true`
3. Run: `mvn -e -Ddependency-check.skip=true verify`
4. Verify that all tests pass

## Risk

### Level 

`Low`

### Required Testing

`Smoke`, `Regression`

### Risk Summary

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
